### ISSUE TITLE
Retroarch can't launch with Onion V4.2.0-beta-dev

### ISSUE THREAD

**Issue by dreams897** [OnionUI/Onion#937]
### Checklist

- [X] I am reporting a bug in the Onion OS (Note that Onion devs can’t fix inherited issues with RA Cores)
- [X] I have reviewed the [Onion Wiki](https://github.com/OnionUI/Onion/wiki) and did not find the answer
- [X] I have searched the existing (open and closed) issues to ensure I do not log a duplicate

### Onions OS Version

Onion V4.2.0-beta-dev

### Miyoo Firmware Version

202303262339

### Provide a clear and concise description of the issue

I have tried 3 times now and installing the latest Onion V4.2.0-beta-dev on Miyoo Plus seems to work but after that I cannot launch Retro Arch. It attempts to open and before the UI is displayed it crashes and exits to the app menu. 

### Steps to Reproduce

Download latest Onion V4.2.0-beta-dev, extract files to microSD formatted using guiformat and go through installation. When complete try to launch Retroarch

### Provide any additional context or information that may help us investigate your issue

NA

**Comment by Aemiii91** [comment]
Duplicate of #912
